src/uwtools/drivers/driver.py

```diff
@@ -340,14 +340,14 @@ def _validate(self) -> None:
         for schema_name in (self._driver_name.replace("_", "-"), "platform"):
             validate_internal(schema_name=schema_name, config=self._config)
 
-    def _write_runscript(self, path: Path, envvars: Dict[str, str]) -> None:
+    def _write_runscript(self, path: Path, envvars: Optional[Dict[str, str]] = None) -> None:
```

Comment:
Many drivers were passing in a `envvars={}` argument, which seemed unnecessarily noisy to me, so I made this optional, supplied the `{}` default value below, and simplified the calls to this method where possible.